### PR TITLE
Portfolio 3

### PR DESCRIPTION
**Pull request by dmunozc**
Major Changes:
- Added iexfinance as a dependency to query data from IEX
- Added news feed for portfolio page
- Added 'invesment sector allocation' pie chart to portfolio page
- Tried to make our main page more responsive to screen changes
  - I collapsed our sidebar by default since it was causing problems on mobile screens

### CODE REVIEW

File: templates/layout.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
 <body>
   <div id="wrapper">
     <!-- Navigation Bar (Side) -->
-    <nav id="sidebar" class="navbar-default navbar-static-side">
+    <nav id="sidebar" class="navbar-default navbar-static-side toggled active">
```
Comment (carissaallen) [inline templates/layout.html:43]: Great idea! 👍

**Review by carissaallen — APPROVED** [review]
Great! I'm happy to approve, but am wondering what this middle section is? 

Screen Shot 2020-03-06 at 5 37 22 PM

**Comment by dmunozc** [comment]
Hey Carissa, there is supposed to be a pie graph there
Screenshot_2020-03-06_18-13-30

Can you try loading the page in an incognito window?

**Comment by dmunozc** [comment]
Going to first merge PR 19 and then going to create a new PR for these changes

**Pull request closed without merging**